pkg/resources/account_password_policy_attachment_acceptance_test.go

```diff
@@ -43,26 +43,15 @@ func TestAcc_AccountPasswordPolicyAttachment(t *testing.T) {
 
 func accountPasswordPolicyAttachmentConfig(prefix string) string {
 	s := `
-resource "snowflake_database" "test" {
-	name = "%v"
-	comment = "Terraform acceptance test"
-}
-
-resource "snowflake_schema" "test" {
-	name = "%v"
-	database = snowflake_database.test.name
-	comment = "Terraform acceptance test"
-	}
-
 resource "snowflake_password_policy" "pa" {
-	database   = snowflake_database.test.name
-	schema     = snowflake_schema.test.name
+	database   = "terraform_test_database"
```

Comment:
We have this `terraform_test_database` extracted as constant `TestDatabaseName.` I would prefer having here `%s` and filling it in the Sprintf at the bottom. It is more fault-proof if, for some reason, we move to `terraform_test_database_<some_uuid>` (like we are doing in the SDK integration tests).

It goes for every usage of these strings instead of our constants.